packages/mdc-dialog/component.ts

```diff
@@ -121,15 +123,17 @@ export class MDCDialog extends MDCComponent<MDCDialogFoundation> {
       document.removeEventListener('keydown', this.handleDocumentKeydown_);
     };
 
-    this.listen('click', this.handleInteraction_);
-    this.listen('keydown', this.handleInteraction_);
+    this.listen('click', this.handleClick_);
+    this.listen('keydown', this.handleKeydown_);
+    this.listen('touchmove', this.handleOverScroll_);
```

Comment:
We need to attach this event only to `scrim` element. Then this method can be named as `handleScrimTouchmove`.

Reply:
I tried that it doesn't work if we add this only to scrim since it's not a parent of dialog. so overscrolling dialog will scroll the elements on back.